src/main/resources/i18n/aerospace/common.properties

```diff
@@ -292,6 +292,8 @@ SUBMIT_ERROR=Failed to submit this order.
 SUBMIT_ERROR_TITLE=Failed to submit the order
 KAFKA_BROKER_NOT_AVAILABLE=Could not establish connection to Kafka, broker may not be available.
 RABBITMQ_HOST_NOT_AVAILABLE=Could not establish connection to RabbitMQ, broker may not be available.
+ORDER_CONTAINS_INVALID_ITEMS=Order contains invalid items.
+NOT_ENOUGH_INVENTORY_TO_FULFILL_ORDER=There is not enough inventory to fulfill your order.
```

Comment:
Let's change this to be "Order contains items which are out of stock."
And also use the message in the order detail modal dialog to be consistent.
![image](https://user-images.githubusercontent.com/35082985/212165857-ec6586e3-d0b6-4f2d-a4d1-392cf726bac6.png)